artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -119,5 +119,6 @@ exec "$JAVACMD" \
     -Ddata.dir="$ARTEMIS_DATA_DIR" \
     -Djava.util.logging.manager="$ARTEMIS_LOG_MANAGER" \
     -Dlogging.configuration="$ARTEMIS_LOGGING_CONF" \
+    -Dhawtio.offline="true" \
```

Comment:
Is this needed as it's set in jvm args in profile with all the other hawtio flags